src/main/java/uk/gov/hmcts/reform/sscs/service/HearingsService.java

```diff
@@ -27,6 +29,10 @@ public class HearingsService {
 
     private final IdamService idamService;
 
+    public void processHearingRequest(HearingRequest hearingRequest) throws UnhandleableHearingStateException, GetCaseException {
+        processHearingRequest(createWrapper(hearingRequest));
+    }
+
     public void processHearingRequest(HearingWrapper wrapper) throws UnhandleableHearingStateException {
```

Comment:
Why don't we just call createWrapper in here? - don't think we need a separate method for this, harder to follow as well.